lib/tasks/create_staging_data.rake

```diff
@@ -154,10 +154,26 @@ staging = namespace :staging do
       :category => :bundle,
       :subscription => true,
       :price => 70.00,
-      :name => 'Season Subscription',
+      :name => 'Season Subscription 1',
       :season => Time.this_season,
       :offer_public => Vouchertype::ANYONE,
       :included_vouchers => sub_vouchers.map(&:id).map(&:to_s).zip(Array.new(sub_vouchers.size) { 1 }).to_h)
+    sub = Vouchertype.create!(
+        :category => :bundle,
+        :subscription => true,
+        :price => 80.00,
+        :name => 'Season Subscription 2',
+        :season => Time.this_season,
+        :offer_public => Vouchertype::ANYONE,
+        :included_vouchers => sub_vouchers.map(&:id).map(&:to_s).zip(Array.new(sub_vouchers.size) { 1 }).to_h)
+    sub = Vouchertype.create!(
```

Comment:
why do you assign the same variable twice?  if you're not going to use the variable don't assign it.  this is confusing usage.  if all you're doing is creating more sub vouchertypes you should put the creation code in a loop that just changes the name each time - please be DRY, do not copy-and-paste code.

Reply:
Sorry for the repetition here, I have reverted reate_staging_data.rake back to its original version. I will open another feature branch for adding more subscription_vouchertypes.